File: Data_Structures/JAVA/src/00Array/Array0.java
public class Array0 {

    //time and space complexity = 0(n2)T  0(1)S
    //To find the two numbers in the array to get the target value
    public static int[] twoNumberSum(int[] array,int target){
        for(int i=0; i<array.length-1; i++){
            for(int j=i+1; j<array.length-1; j++){
                if(array[i]+array[j]==target){
                    return new int[]{array[i],array[j]};
                }
            }
        }
        return new int[0];
    }
    public static void main(String[] args) throws Exception {
int[] array = {6,5,7,9,4,0,2};
int[] result = twoNumberSum(array, 10);
         System.out.println("first:"+result[0]+"second:"+result[1]);
    }
}

File: Data_Structures/JavaScript/00Arrays/array0.js
//user enters a target number, we have to find two numbers from array
// the sum of the numbers should be equal to the target number

const sumOfTwoNumbers = (array,target)=>{
    let newArray = [];
    for(let i= 0;i <=array.length; i++){
        for(j=0; j<=array.length; j++){
            if(array[i] == array[j]){
                continue;
            }else if(array[i]+ array[j] == target){
                newArray.push(array[i]);
                newArray.push(array[j]);
                return newArray;
               
            }
        }
    }

}

const array = [6, 5, 13, 4, 9, 0]
let result = sumOfTwoNumbers(array,10);
console.log(result);

//Here we have used two nested array ,so the time complexiety is O(n^2)